### DISCUSSION TITLE
how to use allowReorder and submit the images

### DISCUSSION THREAD

**Discussion by allaghi** [rahulhaque/laravel-filepond#35] [Q&A]
Hi
have a nice day,

I am wondering if is there any a proper way to submit reordered images, when I use allowReorder and set it true I can reorder them but once I submit them they are not .

**Note I use vue version with inerita.**

Thanks in advance.

**Comment by rahulhaque** [comment]
@allaghi file reordering supposed to be done from frontend. My package will process the files maintaining the order they're sent from the frontend. If filepond supports reordering, it should reorder the inputs it appends in the form. There's no official documentation currently available for reordering files on filepond. I won't guess before any stable release.
